WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInput.swift

```diff
@@ -122,17 +174,31 @@ private extension ShippingLabelCustomsFormInput {
         static let contentExplanationPlaceholder = NSLocalizedString("Type of contents",
                                                                      comment: "Placeholder for the Content Details row " +
                                                                         "in Customs screen of Shipping Label flow")
+        static let contentExplanationMissing = NSLocalizedString("Please describe what kind of goods this package contains",
+                                                                 comment: "Error message for missing explanation when Content Type" +
+                                                                    "is Other in Customs screen of Shipping Label flow")
         static let restrictionTypeTitle = NSLocalizedString("Restriction Type",
                                                             comment: "Title for the Restriction Type row in Customs screen of Shipping Label flow")
         static let restrictionCommentTitle = NSLocalizedString("Restriction Details",
                                                                comment: "Title for the Restriction Details row in Customs screen of Shipping Label flow")
         static let restrictionCommentPlaceholder = NSLocalizedString("Type of restriction",
                                                                      comment: "Placeholder for the Restriction Details row in " +
                                                                         "Customs screen of Shipping Label flow")
+        static let restrictionCommentMissing = NSLocalizedString("Please describe what kind of restrictions this package must have",
+                                                                 comment: "Error message for missing comments when Restriction Type" +
+                                                                    "is Other in Customs screen of Shipping Label flow")
         static let itnTitle = NSLocalizedString("ITN",
                                                 comment: "Title for the ITN row in Customs screen of Shipping Label flow")
         static let itnPlaceholder = NSLocalizedString("Enter ITN (Optional)",
                                                       comment: "Placeholder for the ITN row in Customs screen of Shippling Label flow")
+        static let itnMissingForDestination = NSLocalizedString("ITN is required for shipments to %1$@",
+                                                                comment: "Error message for missing ITN for destination country in" +
+                                                                    "Customs screen of Shipping Label flow")
```

Comment:
nit: It might be helpful to include a brief explanation about the placeholder in the comment (for translators), e.g. "The placeholder is the destination country"